terragrunt/aws/database/security_groups.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_security_group" "cra_upd_docdb_sg" {
+  name        = "cra-upd-docdb-sg"
+  description = "Ingress to DocumentDB Security Group"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description     = "Allow ingress to DocumentDB"
+    from_port       = 27017
+    to_port         = 27017
+    protocol        = "tcp"
+    security_groups = [aws_security_group.cra_upd_docdb_egress_sg.id]
+  }
+}
+
+resource "aws_security_group" "cra_upd_docdb_egress_sg" {
```

Comment:
Does this actually allow egress to the Document DB? It seems that there are no specific rules and it seems that this is just a container that does not enforce any rules, unless I am missing something.

Reply:
Yes, it's an empty security group that doesn't allow anything on its own, but the DocumentDB's security group is set up to only allows ingress from resources that are assigned this specific security group. So you explicitly add that security group to allow connecting to it. I use the same pattern for a few other resources that will be added later as well.

Reply:
Sounds good - thanks for the clarification.